### PR TITLE
Accessibility for automation

### PR DESCRIPTION
**Pull request by captainamiedi**
Fixes #
Accessibility for automation 

Changes in this pull request:
- added testID  attribute to TouchableOpacity
- added accessible attribute to TouchaleOpacity
- added accessibleLabel attribute to TouchableOpacity

@

### CODE REVIEW

**Comment by sagararyal** [comment]
Screen Shot 2020-08-04 at 6 40 03 PM

Screen Shot 2020-08-04 at 6 40 41 PM

File: .github/workflows/macos_build_ios.yml [hunk 1]
```diff
@@ -147,7 +148,7 @@ jobs:
       env:
         BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
       run: |
-        curl -u "planetit1:$BROWSERSTACK_ACCESS_KEY" -X POST "https://api-cloud.browserstack.com/app-live/upload" -F "file=@uploads/$upload_file"
```
Comment (norbertschuler) [inline .github/workflows/macos_build_ios.yml:150]: Don't we want to be able to test the Treemapper app anymore with BrowserStack App Live?

Comment (sagararyal) [reply .github/workflows/macos_build_ios.yml:150]: We can still upload the builds from drive. 

App automate can run automation based on the tests we write. Locally or on browser stack web.
I would say this is an upgrade.

Comment (norbertschuler) [reply .github/workflows/macos_build_ios.yml:150]: Aren't we allowed to upload the files to both services at BrowserStack using two curl commands?

Comment (sagararyal) [reply .github/workflows/macos_build_ios.yml:150]: We can definitely do that.

File: .github/workflows/macos_build_ios.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches:
       - develop
+      - feature/tree_inventory
```
Comment (norbertschuler) [inline .github/workflows/macos_build_ios.yml:10]: Please remove this again before this branch gets merged.

File: .github/workflows/ubuntu_build_android.yml [hunk 3]
```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches:
       - develop
+      - feature/tree_inventory
```
Comment (norbertschuler) [inline .github/workflows/ubuntu_build_android.yml:10]: Please remove this again before this branch gets merged.

File: .github/workflows/ubuntu_build_test.yml [hunk 4]
```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches:
       - develop
+      - feature/tree_inventory
```
Comment (norbertschuler) [inline .github/workflows/ubuntu_build_test.yml:10]: Please remove this again before this branch gets merged.

File: .github/workflows/macos_build_ios.yml (hunk 2 above)
Comment (captainamiedi) [reply .github/workflows/macos_build_ios.yml:10]: These have been removed

File: .github/workflows/ubuntu_build_android.yml (hunk 3 above)
Comment (captainamiedi) [reply .github/workflows/ubuntu_build_android.yml:10]: these has also been removed

File: .github/workflows/ubuntu_build_test.yml (hunk 4 above)
Comment (captainamiedi) [reply .github/workflows/ubuntu_build_test.yml:10]: these has been removed as well

**Pull request merged**